organization-plan.md

```diff
@@ -3,32 +3,23 @@
 This npm Organization Payment Plan (this _Payment Plan_) supplements
 the terms for npm Open Source offered by npm, Inc. (_npm_) at
 <https://www.npmjs.com/policies/open-source-terms> (_npm Open Source
-Terms_), as well as the terms for npm Private Packages (_npm Private
-Packages_) at <https://www.npmjs.com/policies/private-terms> (_npm
-Private Packages Terms_). This Payment Plan governs payment for npm
-Private Packages organizations (_Organizations_) and use of npm Private
-Packages by user accounts added as members of those Organizations.
+Terms_), as well as the terms for npm Paid Services (_npm Paid Services)
+at <https://www.npmjs.com/policies/private-terms> (_npm
+Paid Services Terms_). This Payment Plan governs payment for
+(_npm Organizations_) and use of npm Paid Services by user
+accounts added as members of those Organizations.
 
 This Payment Plan was last updated on
-December 1, 2015.
+June 27, 2017.
 You can review prior versions at
 <https://github.com/npm/policies/commits/master/organization-plan.md>.
 
 Under this Payment Plan, you may create one or more Organizations.
 
-You will pay a minimum of $14.00 via your Payment Card when you create
-an Organization and at the end of each month-long period, until you
-delete the organization. This minimum payment entitles you to add up to
-two user accounts not otherwise entitled to use npm Private Packages as
-member of the organization (_New Private Packages Accounts_), and an
-unlimited number of other user accounts. You will pay $7.00 via your
-Payment Card per each additional New Private Package User (above two)
-that you add to an Organization, when you add them as a member and at
-the end of each month-long period.
-
-The account holders of New Private Packages Accounts that you add as
-members of your Organizations will be entitled to use npm Private
-Packages under this Payment Plan, provided the account holders agree to
-and abide by the npm Open Source Terms and npm Private Packages Terms.
-These rights will not limit npm's right to enforce the npm Open Source
-Terms, npm Private Packages Terms, or other terms.
+You will pay a minimum of $7.00 via your Payment Card when you create
+an Organization and thereafter on the same day every month (your
+_Billing Day_), until you delete the organization. This minimum payment
+entitles you to a single member of the organization (a _New Paid Services
+User_). You will pay $7.00 via your Payment Card per each additional
+New Paid Services User that you add to an Organization, counted and
+billed on your Billing Day.
```

Comment:
Let's take pricing out of the terms, and incorporate from the site. Is there a stable URL we can reference for pricing?